src/lev-report/ListFilter.js

```diff
@@ -0,0 +1,28 @@
+ import React from 'react';
+import styled from 'styled-components';
+import { InputField } from 'govuk-react';
+
+const StyledListFilter = styled(InputField)`
+  display: inline-block;
+  max-width: 15em;
+  width: 15em;
+  margin-right: 2em;
+`;
+
+const ListFilter = ({ hint, children, input}) =>
+<StyledListFilter
+    hint={hint}
+    input={{
+        ...input,
+        list: "list"
+    }}
+    >
+    <datalist id="list">
```

Comment:
`list` could clash quite easily.

Perhaps `group-filter` would be a bit better?

Do we need an id set at all?

Reply:
Fixed. id is needed to associate list with input